chore: align naming between instance properties and operations (#3127)
SAP/cloud-sdk-js: V3-Upgrade-Guide.md

Running your tests or deploying your application might fail at this point in time if you need to adapt to any breaking changes.
We recommend updating your applications in one commit or pull request and making sure everything still works using your existing test suite.

> ### Adjust Operation Names in Odata Generated Clients

Reviewer: ```suggestion
### Adjust Operation Names in OData Generated Clients
```

they spell it with a capital d, cf https://www.odata.org/

> Rules for naming of functions (bound or otherwise) in the generated client have been changed.
> If a function/actions begins with an `underscore` symbol, the `_` will be removed from the resulting generated client code.
> To adjust the names, search in `function-import.ts` and `action-import.ts` files for any operation starting with `_`.

Reviewer: ```suggestion
Rules for naming of OData operations (actions or functions) in the generated client have been changed.
This applies to bund and unbound operations.
If an operation begins with an `underscore` symbol, the `_` will be removed from the resulting generated client code.
To adjust the names, search in `function-import.ts` and `action-import.ts` files for any operation starting with `_`.
```

Reviewer: rephrased a bit, hope that makes it more clear
